src/Foundatio/Storage/FolderFileStorage.cs

```diff
@@ -7,15 +7,19 @@
 using Foundatio.Extensions;
 using Foundatio.Serializer;
 using Foundatio.Utility;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 
 namespace Foundatio.Storage {
     public class FolderFileStorage : IFileStorage {
         private readonly object _lockObject = new object();
         private readonly ISerializer _serializer;
+        protected readonly ILogger _logger;
 
-        public FolderFileStorage(string folder, ISerializer serializer = null) {
+        public FolderFileStorage(string folder, ISerializer serializer = null, ILoggerFactory loggerFactory = null) {
```

Comment:
We should create a Options class like the Queues, Cache and Message buses have to be consistent here.